common-utils/src/main/java/eu/arrowhead/common/http/filter/authorization/BlacklistFilter.java

```diff
@@ -43,42 +56,87 @@ public class BlacklistFilter extends ArrowheadFilter {
 
 	//-------------------------------------------------------------------------------------------------
 	@Override
-	protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response, final FilterChain chain) throws IOException, ServletException {	
+	protected void doFilterInternal(final HttpServletRequest request, final HttpServletResponse response, final FilterChain chain) throws IOException, ServletException {
 		log.debug("BlacklistFilter is active");
-		
+
+        final MultiReadRequestWrapper requestWrapper = new MultiReadRequestWrapper((HttpServletRequest) request);
+
 		// if requester is sysop, no need for check
-		final boolean isSysop = Boolean.valueOf(request.getAttribute(Constants.HTTP_ATTR_ARROWHEAD_SYSOP_REQUEST).toString());
-		
+		final boolean isSysop = HttpUtilities.isSysop(requestWrapper, "BlacklistFilter.doFilterInternal");
+
 		// if request is for lookup for authentication, no need for check
-		final boolean isAuthLookup = false; //TODO
-		
+		final boolean isAuthLookup = isAuthLookup(requestWrapper);
```

Comment:
Please rename the variable and method to isAuthenticationLookup. Auth could mean Authorization as well, and it will be confusing later.